Linking to demo and websites (#26)
ably-labs/atomic-examples: src/components/AvatarStack/README.md

## Resources

> - Read further more about building an [avatar stack](https://ably.com/examples/avatar-stack?utm_source=ably-labs&utm_medium=github&utm_campaign=avatar-stack).

Reviewer: ```suggestion
- Learn more about how to build your own [avatar stack](https://ably.com/examples/avatar-stack?utm_source=ably-labs&utm_medium=github&utm_campaign=avatar-stack).
```

> - Read further about what is possible with an [avatar stack](https://ably.com/examples/avatar-stack?utm_source=ably-labs&utm_medium=github&utm_campaign=avatar-stack).

Reviewer: Can we use 'what is possible' in just one of these sentences & reword the other one @ginotoro

Author: Updated

Reviewer: Made another suggestion! :)
